homeassistant/components/sensor/airvisual.py

```diff
@@ -217,14 +233,16 @@ class AirQualityIndexSensor(AirVisualBaseSensor):
     @property
     def unit_of_measurement(self):
         """Return the unit the value is expressed in."""
-        return ''
+        return 'unit' if self.state == '1' else 'units'
```

Comment:
Missed this one first pass. I don't think it's wise to change the unit for the same entity. That will mess with 3rd party recorders and be very unpredictable for the users. Just decide on either of these.